src/ResourceTemplate/RelativeResourceTemplate.php

```diff
@@ -272,12 +272,15 @@ public function match($path)
         return $collapsedBindings;
     }
 
-    private function matchException($path, $reason)
+    /**
+     * @param string|null $path
+     */
+    private function matchException($path, string $reason)
     {
         return new ValidationException("Could not match path '$path' to template '$this': $reason");
     }
 
-    private function renderingException($bindings, $reason)
+    private function renderingException(array $bindings, string $reason)
```

Comment:
Nit: php doc?

Reply:
My general rule with this PR was to only add `phpdoc` if I couldn't typehint the parameters properly

Reply:
Yeah that's fair.